home/views/api/dailywalk.py

```diff
@@ -116,7 +117,38 @@ def post(self, request, *args, **kwargs):
                 }
             )
 
-        # Associate contest with user (account)
+            # total_steps = DailyWalk.objects.get
```

Comment:
I think you need to pull the indentation back one level less, otherwise this is inside the loop...?